packages/trie/README.md

```diff
@@ -181,76 +208,89 @@ async function test() {
 
-- `random.ts` runs random `PUT` operations on the tree.
-- `checkpointing.ts` runs checkpoints and commits between `PUT` operations.
+```shell
+npm run test:browser
+```
 
-A third benchmark using mainnet data to simulate real load is also under consideration.
+> Note that this requires an installation of [Mozilla Firefox](https://www.mozilla.org/en-US/firefox/new/), otherwise the tests will fail.
 
-Benchmarks can be run with:
+You may run tests for node.js using:
+
+```shell
+npm run test:node
+```
+
+## Benchmarking
+
+You will find two simple **benchmarks** in the `benchmarks` folder:
+
+- `random.ts` runs random `PUT` operations on the tree, and
+- `checkpointing.ts` runs checkpoints and commits between `PUT` operations
+
+A third benchmark using mainnet data to simulate real load is also being considered.
+
+You may run benchmarks using:
 
 ```shell
 npm run benchmarks
 ```
 
-To run a **profiler** on the `random.ts` benchmark and generate a flamegraph with [0x](https://github.com/davidmarkclements/0x) you can use:
+To run a **profiler** on the `random.ts` benchmark and generate a flamegraph with [0x](https://github.com/davidmarkclements/0x), you may use:
 
 ```shell
 npm run profiling
 ```
 
 0x processes the stacks and generates a profile folder (`<pid>.0x`) containing [`flamegraph.html`](https://github.com/davidmarkclements/0x/blob/master/docs/ui.md).
 
-# REFERENCES
+## References
 
 - Wiki
   - [Ethereum Trie Specification](https://github.com/ethereum/wiki/wiki/Patricia-Tree)
 - Blog posts
   - [Ethereum's Merkle Patricia Trees - An Interactive JavaScript Tutorial](https://rockwaterweb.com/ethereum-merkle-patricia-trees-javascript-tutorial/)
   - [Merkling in Ethereum](https://blog.ethereum.org/2015/11/15/merkling-in-ethereum/)
-  - [Understanding the Ethereum Trie](https://easythereentropy.wordpress.com/2014/06/04/understanding-the-ethereum-trie/). Worth a read, but the Python libraries are outdated.
+  - [Understanding the Ethereum Trie](https://easythereentropy.wordpress.com/2014/06/04/understanding-the-ethereum-trie/) (This is worth reading, but mind the outdated Python libraries)
 - Videos
   - [Trie and Patricia Trie Overview](https://www.youtube.com/watch?v=jXAHLqQthKw&t=26s)
 
-# EthereumJS
-
-See our organizational [documentation](https://ethereumjs.readthedocs.io) for an introduction to `EthereumJS` as well as information on current standards and best practices.
+## EthereumJS
 
-If you want to join for work or do improvements on the libraries have a look at our [contribution guidelines](https://ethereumjs.readthedocs.io/en/latest/contributing.html).
+See our organizational [documentation](https://ethereumjs.readthedocs.io) for an introduction to `EthereumJS` as well as information on current standards and best practices. If you want to join for work or carry out improvements on the libraries, please review our [contribution guidelines](https://ethereumjs.readthedocs.io/en/latest/contributing.html) first.
```

Comment:
This is also a generic section below all READMEs for all packages, if you do changes here you should "broadcast" to the other packages as well.